Assets/Plugins/Editor/Uplift/Windows/SettingsWindow.cs

```diff
@@ -44,7 +44,11 @@ protected void OnGUI()
 #endif
             scrollPosition = EditorGUILayout.BeginScrollView(scrollPosition);
 
-            if (!File.Exists(UpliftSettings.GetDefaultLocation()))
+            string SettingsLocation = UpliftSettings.GetDefaultLocation();
```

Comment:
Maybe `settingsLocation` ? It's the convention we used so far.

Reply:
Updated.